src/EditorFeatures/Core.Wpf/Suggestions/FixAll/FixMultipleOccurrencesService.cs

```diff
@@ -22,25 +24,28 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
     /// Service to compute and apply <see cref="FixMultipleCodeAction"/> code fixes.
     /// </summary>
     [ExportWorkspaceService(typeof(IFixMultipleOccurrencesService), ServiceLayer.Host), Shared]
-    internal class FixMultipleOccurrencesService : IFixMultipleOccurrencesService
+    internal sealed class FixMultipleOccurrencesService : IFixMultipleOccurrencesService
     {
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public FixMultipleOccurrencesService(IAsynchronousOperationListenerProvider listenerProvider)
-            => listenerProvider.GetListener(FeatureAttribute.LightBulb);
+        public FixMultipleOccurrencesService(IGlobalOptionService globalOptions, IAsynchronousOperationListenerProvider listenerProvider)
+        {
+            listenerProvider.GetListener(FeatureAttribute.LightBulb);
```

Comment:
😕 What's going on here?

Reply:
Changed the approach a bit and forgot to clean up.

Reply:
![image](https://user-images.githubusercontent.com/41759/154331252-3a76a1af-1668-4776-9986-8381c03ce6ae.png)